conf/deployment/azure/ipi_1az_rhcos_3m_3w.yaml

```diff
@@ -7,10 +7,10 @@ ENV_DATA:
   deployment_type: 'ipi'
   region: 'eastus'
   azure_base_domain_resource_group_name: 'ocsqe'
-  base_domain: 'azure.qe.rh-ocs.com'
+  base_domain: 'azure2.qe.rh-ocs.com'
   worker_replicas: 3
   master_replicas: 3
-  master_instance_type: 'Standard_D8s_v3'
+  master_instance_type: 'Standard_D16s_v3'
```

Comment:
do we also need higher instance type for master instance? I know that in other platforms (AWS) we have m4.xlarge for masters in production configs (the default one) and m5.4xlarge for workers IIRC. So just curious why we also need to change the master instance type now here for azure?  The m4.xlarge has 4 vCPU and 16 GB of memory I think. I found that Standard_D16s_v3 should have about 16 vCPU and 64 GB which is probably worth only for OCS nodes but not sure if we need to pay for that and have this big power for master nodes.

Reply:
Good point. We will need to audit the instance type usage and recommendations for all platforms and configurations.

I recall that for reasons I haven't looked into, we use quite large instances for both master and worker nodes in our OCS clusters, and assumed that we need to update the master node instance for a production like configurations here as well. But there seems to be no requirements for that, so I will just revert that change.

Checking the instance types shows that for azure master node instance, we could go even lower and still match AWS instance type. But for that choice, we don't have any requirement.

Azure:

- Standard_D4_v3: 4 vcpu, 16 GiB ram
- Standard_D8s_v3: 8 vcpu, 32 GiB ram
- Standard_D16s_v3: 16 vcpu, 64 GiB ram

AWS:

- m5.4xlarge: 16 vcpu, 64 GiB ram
- m4.xlarge: 4 vcpu, 16 GiB ram

Sources:
- https://docs.microsoft.com/en-us/azure/virtual-machines/dv3-dsv3-series
- https://www.amazonaws.cn/en/ec2/instance-types/

Reply:
reverted in ead042ac39b4cdce1d55fad9a74c062aaf56a64b